nengo_gui/static/netgraph.js

```diff
@@ -625,3 +640,11 @@ Nengo.NetGraph.prototype.scaleMiniMapViewBox = function () {
     this.view.setAttribute('width', w / this.scale);
     this.view.setAttribute('height', h / this.scale);
 }
+
+Nengo.NetGraph.prototype.get_aspect_resize = function() {
```

Comment:
Hmm..  we should make a general style decision here.  I could definitely see generally making setters/getters even when we're not doing anything, for the standard reasons of making it easier to make them complex in the future.